modules/nw-traditional-sharding.adoc

```diff
@@ -26,7 +26,7 @@ spec:
       - ops
 ----
 
-An example of a configured Ingress Controller `dev-router` that has the label selector `spec.namespaceSelector.matchLabels.name` with the key value set to `dev`:
+An example of a configured Ingress Controller `dev-router` that has the label selector `spec.namespaceSelector.matchLabels.name` with the key value set to `"dev"`:
```

Comment:
Should the phrase "label selector" follow `spec.namespaceSelector.matchLabels.name`? For example, "...that has the `spec.namespaceSelector.matchLabels.name` label selector with the key value set to `"dev"`:"